langchain/package.json

```diff
@@ -1372,6 +1372,7 @@
     "langchainhub": "~0.0.6",
     "langsmith": "~0.0.48",
     "ml-distance": "^4.0.0",
+    "officeparser": "^4.0.4",
```

Comment:
Can you add this as a peer dep + optional dep and mark it as optional under `peerDependenciesMeta`?